server/src/providers/blockchain.ts

```diff
@@ -205,7 +206,7 @@ export class blockchain {
     await this.updateEvents()
 
     const oneTimePaymentLinksAddress: any = get(ContractsAddress, `${this.network}.OneTimePayments`)
-    const inEscrow = await this.tokenContract.methods.balanceOf(oneTimePaymentLinksAddress).call()
+    const inEscrow = await retryTimeout(this.tokenContract.methods.balanceOf(oneTimePaymentLinksAddress).call)
```

Comment:
```suggestion
    const inEscrow = await retryTimeout(() => this.tokenContract.methods.balanceOf(oneTimePaymentLinksAddress).call())
```
let's keep the interface of retryTimeout invoking . let's pass arrow fn wrapping some wallet method everywhere (instead the method itself)